website/docs/docs/cloud/dbt-copilot.md

```diff
@@ -3,25 +3,39 @@ title: "About dbt Copilot"
 
-To use dbt Copilot, you must have an active [dbt Cloud Enterprise account](https://www.getdbt.com/pricing) and either agree to use dbt Labs' OpenAI key or provide your own Open AI API key. [Register here](https://docs.google.com/forms/d/e/1FAIpQLScPjRGyrtgfmdY919Pf3kgqI5E95xxPXz-8JoVruw-L9jVtxg/viewform) or reach out to the Account Team if you're interested in joining the private beta.
+With automatic code generation, let dbt Copilot [generate code](/docs/cloud/use-dbt-copilot) using natural language, and [generate documentation](/docs/build/documentation), [tests](/docs/build/data-tests), [metrics](/docs/build/metrics-overview), and [semantic models](/docs/build/semantic-models) for you with the click of a button in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-copilot) and [Visual Editor](/docs/cloud/build-ve-copilot).
+
+:::tip
+dbt Copilot is available to all Enterprise accounts. If you're interested in using dbt Copilot, [book a demo](https://calendly.com/dbt-labs-sales/dbt-copilot-demo) and we'll help answer any questions you have.
 :::
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-ide/dbt-copilot-doc.gif" width="100%" title="Example of using dbt Copilot to generate documentation in the IDE" />
 
-## Feedback
+## How dbt Copilot works
+
+dbt Copilot enhances efficiency by automating repetitive tasks while ensuring data privacy and security. It works as follows:
 
-Please note: Always review AI-generated code and content as it may produce incorrect results. The features and/or functionality of dbt Copilot may be added or eliminated as part of the beta trial.
+- Access dbt Copilot through the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-copilot) to generate documentation, tests, semantic models, or in the [Visual Editor](/docs/cloud/build-ve-copilot) to generate SQL code using natural language prompts.
+- dbt Copilot gathers metadata (like column names, model SQL, documentation) but never accesses row-level warehouse data.
+- The metadata and user prompts are sent to the AI provider (in this case, OpenAI) through API calls for processing.
+- The AI-generated content is returned to dbt Cloud for you to review, edit, and save within your project files.
+- dbt Copilot does not use warehouse data to train AI models.
+- No sensitive data persists on dbt Labs' systems, except for usage data.
+- You can request the deletion of personal or sensitive data within 30 days.
+
+:::tip
+dbt Copilot accelerates, but doesn’t replace, your analytics development lifecycle. It helps deliver better data products faster, but always review AI-generated content, as it may be incorrect.
+:::
 
-To give feedback, please contact your dbt Labs account team. We appreciate your feedback and suggestions as we improve dbt Copilot.
+WE CAN ADD ARCHITECTURAL DIAGRAM HERE
```

Comment:
waiting for [design diagram](https://app.asana.com/1/312418853397011/project/1202263237307725/task/1209641484239277?focus=true)